app/lib/bk/compat/parsers/circleci.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 require_relative '../translator'
-require_relative '../pipeline'
+require_relative '../models/pipeline'
+require_relative '../models/steps/circleci/circleci_step'
+
```

Comment:
This file does not appear to be using pipelines or its specific command step any more, but only group steps::

```suggestion
require_relative '../../models/steps/group'

```

Reply:
Adjusted 👍